tensorboard/webapp/plugins/plugins_container_test.ts

```diff
@@ -359,6 +345,46 @@ describe('plugins_component', () => {
     });
   });
 
+  describe('IFRAME plugin DOM creation', () => {
+    it('creates an element for IFRAME type of plugin', async () => {
+      configureMockStore();
+      const pluginApiHost = TestBed.inject(PluginApiHostModule);
+      const registerPluginIframeSpy = spyOn(
+        pluginApiHost,
+        'registerPluginIframe'
+      );
+      const fixture = TestBed.createComponent(PluginsContainer);
+      fixture.detectChanges();
+
+      setActivePlugin('foo');
+
+      fixture.detectChanges();
+      await fixture.whenStable();
+
+      const {nativeElement} = fixture.debugElement.query(By.css('.plugins'));
+      expect(nativeElement.childElementCount).toBe(1);
+      const pluginElement = nativeElement.children[0];
+      expectPluginIframe(pluginElement, 'foo');
+      expect(registerPluginIframeSpy).toHaveBeenCalledWith(
+        pluginElement,
+        'foo'
+      );
+    });
+
+    it('throws error for IFRAME type of plugin when no PluginApiHostModule', async () => {
+      TestBed.overrideProvider(PluginApiHostModule, {useValue: null});
+      configureMockStore();
```

Comment:
Is there any importance to this order? If not, was `configureMockStore` refactor required?

Reply:
Yes, order is important. I tried to document this effectively in configureMockStore() but also have now added a comment here.